bastion.tf

```diff
@@ -25,11 +28,6 @@ resource "google_compute_instance" "bastion" {
     enable-oslogin         = "true"
   }
 
-  service_account {
```

Comment:
Why have we removed this @sjmiller609 ?

GCE Instance would use the default service account if we don't specify a service account.